tools/go-cli/go-whisk-cli/commands/namespace.go

```diff
@@ -41,6 +41,10 @@ var namespaceListCmd = &cobra.Command{
     RunE: func(cmd *cobra.Command, args []string) error {
         // add "TYPE" --> public / private
 
+        if whiskErr := checkArgs(args, 0, 0, "Namespace list", ""); whiskErr != nil {
```

Comment:
The error message would be provided in case args were accidentally provided.  ie.. "Invalid argument(s): X, Y.  The command .... takes no arguments."